.travis.yml

```diff
@@ -0,0 +1,10 @@
+dist: trusty
```

Comment:
Can you try to go to the new style of travis files we use in RIOT master?

https://github.com/RIOT-OS/RIOT/blob/eb6d9b052f6dc65ccf52b3d5a4eea35e54eb76b3/.travis.yml#L1-L14

Reply:
sure